src/main/java/org/apache/maven/plugins/dependency/resolvers/GoOfflineMojo.java

```diff
@@ -152,6 +169,46 @@ protected Set<Artifact> resolvePluginArtifacts() throws DependencyResolverExcept
         return resolveDependableCoordinate(buildingRequest, dependableCoordinates, "plugins");
     }
 
+    protected FilterArtifacts getArtifactsFilter() {
+        final FilterArtifacts filter = new FilterArtifacts();
+
```

Comment:
We have the same method in `ResolvePluginsMojo` - we can move it to `AbstractDependencyFilterMojo`